argm.c

```diff
@@ -311,17 +311,25 @@ argm_combine_universal(PG_FUNCTION_ARGS, int32 compareFunctionResultToAdvance)
 
     GET_AGG_CONTEXT(fcinfo, aggcontext);
 
-    if (PG_ARGISNULL(1))
-        PG_RETURN_POINTER(PG_GETARG_POINTER(0));
```

Comment:
This was the bug: if 0-th argument was an SQL null too, we would return an uninitialized pointer here. It's probably always 0, but still leads to segmentation fault when dereferenced, see below.